serverless/aws/provider.py

```diff
@@ -150,7 +150,7 @@ class Provider(BaseProvider, yaml.YAMLObject):
     yaml_tag = "!Provider"
 
     def __init__(
-        self, runtime=Runtime.PYTHON_3_8, extra_tags=None, timeout=10, stage="development", environment=None, **kwargs
+        self, runtime=Runtime.PYTHON_3_8, extra_tags=None, timeout=60, stage="${opt:stage}", environment=None, **kwargs
```

Comment:
I don't think @mprzytulski meant to change default `timeout`, but only `stage` : >

Reply:
definitely only stage.